flink-libraries/flink-table/pom.xml

```diff
@@ -126,14 +125,15 @@ under the License.
 			<plugin>
 				<groupId>net.alchim31.maven</groupId>
 				<artifactId>scala-maven-plugin</artifactId>
-				<version>3.1.4</version>
+				<version>3.2.2</version>
```

Comment:
I used the newest library `flink-cep-scala` as reference. We are already using different plugin version. I will introduce a global variable for all projects.